### PR TITLE
feat: update breadcrumb style

### PR DESCRIPTION
**Pull request by bddjong**
Updated based on figma

image

### CODE REVIEW

File: components/breadcrumb-nav/css/_mixin.scss [hunk 1]
```diff
@@ -13,10 +13,17 @@
 @mixin utrecht-breadcrumb-nav {
   --utrecht-focus-background-color: var(--utrecht-breadcrumb-nav-link-focus-background-color);
   --utrecht-link-background-color: var(--utrecht-breadcrumb-nav-link-background-color);
+  --utrecht-link-text-decoration: var(--utrecht-breadcrumb-nav-link-text-decoration);
   --utrecht-link-focus-color: var(--utrecht-breadcrumb-nav-link-focus-color, var(--utrecht-breadcrumb-nav-link-color));
-  --utrecht-link-focus-text-decoration: var(--utrecht-link-text-decoration);
+  --utrecht-link-focus-text-decoration: var(
+    --utrecht-breadcrumb-nav-link-focus-text-decoration,
+    var(--utrecht-breadcrumb-nav-link-text-decoration)
+  );
```
Comment (bddjong) [inline components/breadcrumb-nav/css/_mixin.scss:21]: Wasn't sure about this one and the `--utrecht-link-hover-text-decoration` change below. I'd have preferred hardcoding underline for this component, but that would ruin any customisability.

File: components/breadcrumb-nav/css/_mixin.scss [hunk 2]
```diff
@@ -55,6 +62,7 @@
 
 @mixin utrecht-breadcrumb-nav__link--current {
   --utrecht-link-current-font-weight: var(--utrecht-breadcrumb-nav-link-current-font-weight, inherit);
+  --utrecht-link-hover-text-decoration: none;
```
Comment (Robbert) [inline components/breadcrumb-nav/css/_mixin.scss:65]: Misschien is deze style niet nodig. Je kunt `current` samen met `disabled` gebruiken, en `disabled` links hebben sowieso geen interactieve states zoals hover.

File: components/breadcrumb-nav/css/_mixin.scss [hunk 3]
```diff
@@ -13,10 +13,17 @@
...
+  --utrecht-link-focus-text-decoration: var(
+    --utrecht-breadcrumb-nav-link-focus-text-decoration,
+    var(--utrecht-breadcrumb-nav-link-text-decoration)
+  );
   --utrecht-link-hover-color: var(--utrecht-breadcrumb-nav-link-color);
-  --utrecht-link-hover-text-decoration: var(--utrecht-link-text-decoration);
+  --utrecht-link-hover-text-decoration: var(
+    --utrecht-breadcrumb-nav-link-focus-text-decoration,
+    var(--utrecht-breadcrumb-nav-link-text-decoration)
+  );
```
Comment (Robbert) [inline components/breadcrumb-nav/css/_mixin.scss:26]: Is het logisch om `focus` aan `hover` te assignen? Foutje?

File: components/breadcrumb-nav/css/_mixin.scss (hunk 1 above)
Comment (Robbert) [reply components/breadcrumb-nav/css/_mixin.scss:21]: Mentioned below too, but `disabled` already provides to option to limit the styles for the last item of the breadcrumb.

**Review by Robbert — CHANGES_REQUESTED** [review]

**Comment by bddjong** [comment]
Sounds like this might still need a change to make keep the breadcrumb links underlined, but I'd like to merge this anyway so we can update the breadcrumb in VTH and amend later.

**Review by Robbert — APPROVED** [review]

**Pull request merged**